XenAdmin/SettingsPanels/NRPEEditPage.cs

```diff
@@ -75,13 +77,12 @@ public string SubText
             }
         }
 
-        public Image Image => Images.StaticImages._000_EnablePowerControl_h32bit_16;
+        public Image Image => Images.StaticImages._000_Module_h32bit_16;
 
-        public NRPEEditPage(bool isHost)
+        public NRPEEditPage()
         {
-            IsHost = isHost;
             InitializeComponent();
-            Text = "NRPE";
+            Text = Messages.NRPE;

```

Comment:
User visible strings should never be hardcoded in the .cs file (even if the text is not translatable in other languages, because in future it may change to something translatable and then we have a localisation bug).